examples/nextjs/pages/_app.tsx

```diff
@@ -0,0 +1,6 @@
+import { AppProps } from 'next/app'
+import '../styles/index.css'
```

Comment:
Should global.css also be imported here? I am curious to see if with your example you can reproduce https://github.com/vercel/next.js/issues/15218